assets/src/apps/v2/elevator.tsx

```diff
@@ -27,6 +28,7 @@ const TYPE_TO_COMPONENT = {
   evergreen_content: EvergreenContent,
   footer: Footer,
   normal_header: NormalHeader,
+  no_data: NoData,
```

Comment:
There is also a `page_load_no_data` state (see `defaultResponseMapper` in `ScreenContainer`) which we should add here. You should be able to see it briefly when the screen page has first loaded but not yet received widget data from the server. We can just have this one also map to `NoData`.

Reply:
Ah yeah, just wasn't sure if you knew about it. I think it should be okay to have this as the `page_load_no_data` as well, just so we have _something_, and we can revisit it later if desired.